README.md

```diff
@@ -1,7 +1,132 @@
-# Mattermost Boards (Focalboard Plugin)
+# Mattermost Boards
+
+### A self-hosted plugin for managing boards withing a Mattermost installation
+
+Mattermost boards plugins is an open source, multilingual, self-hosted project management tool that's an alternative to Trello, Notion, and Asana.
 
 **[Mattermost Boards](https://mattermost.com/boards/)** is the Mattermost plugin version of Focalboard that combines project management tools with messaging and collaboration for teams of all sizes. To access and use **Mattermost Boards**, install or upgrade to Mattermost v6.0 or later as a [self-hosted server](https://docs.mattermost.com/guides/deployment.html?utm_source=focalboard&utm_campaign=focalboard) or [Cloud server](https://mattermost.com/get-started/?utm_source=focalboard&utm_campaign=focalboard). After logging into Mattermost, select the menu in the top left corner of Mattermost and select **Boards**.
 
-***Mattermost Boards** is installed and enabled by default in Mattermost v6.0 and later.*
+## Try Mattermost Boards Plugin 
+
+### Mattermost Plugin
+
```

Comment:
These two titles seems redundant. @cwarnermm what should we keep ?

Reply:
I agree, @marianunez - no need for the second section header "Mattermost Plugin".

Reply:
Removed `Mattermost Plugin`